chameleon_mailman/tasks.py

```diff
@@ -1,10 +1,38 @@
 from __future__ import absolute_import, unicode_literals
 
-from celery import shared_task
+from datetime import datetime, timedelta
+import logging
+import pytz
 
+from celery import task
+
+from django.conf import settings
 from django.core.mail import send_mail
+from django.utils.html import strip_tags
+from user_news.models import Outage
+
+logger = logging.getLogger(__name__)
+
+
+@task
+def send_outage_reminders(crontab_frequency, send_outage_reminder_before):
+    now = pytz.utc.localize(datetime.utcnow())
```

Comment:
Why is this localizing to the system time? Is the system time not UTC? Is the database TZ not UTC?

I always have to look up how to do this. Apparently "the way" from what I have read is `datetime.now(pytz.utc)`